### DISCUSSION TITLE
Compatibility with fresh.js?

### DISCUSSION THREAD

**Discussion by gutenfries** [alephjs/aleph.js#563] [Ideas] [answered]
Hi all,

It's pretty difficult to talk about web frameworks in the deno world and not mention [fresh](https://fresh.deno.dev).

Coming FROM fresh and having no prior knowledge of aleph, are the two compatible? May I choose to use aleph as my full stack solution, using Fresh as my front end?

If this is not an option, what is the conversion experience like? has anyone tried it?

are there unseen advantages to either framework? (ignoring, of course, the fact that they are both in development)

**Comment by ije** [comment, marked as answer]
nope, they are not compatible. try ultrajs if you more prefer react than preact. unfortunately i don't have time on this project
